From 581e8581e6000a83eb233cfac2221cd303358cfe Mon Sep 17 00:00:00 2001
Message-Id: <581e8581e6000a83eb233cfac2221cd303358cfe.1415245453.git.chang-joon.lee@intel.com>
In-Reply-To: <76203ad7c635d495ef3ec1e07b47bad9db4745ba.1415245452.git.chang-joon.lee@intel.com>
References: <76203ad7c635d495ef3ec1e07b47bad9db4745ba.1415245452.git.chang-joon.lee@intel.com>
From: Deepak M <m.deepak@intel.com>
Date: Fri, 17 Oct 2014 04:44:27 +0530
Subject: [PATCH 30/30] FOR_UPSTREAM [VPG]: pwm: Change initcall sequence for
 pwm driver

The pwm driver initcall sequence is changed from the module_init
to subsys_initcall. This ensures that the PWM driver init is
called before the i915 init, as i915 driver uses the pwm
driver calls for backlight control.

Issue: GMIN-3060
Change-Id: Ief01d0f222c2f3d121facd5f0ca0c8d9e4614135
Signed-off-by: Deepak M <m.deepak@intel.com>
---
 drivers/misc/pwm/pwm_byt_platdrv.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/pwm/pwm_byt_platdrv.c b/drivers/misc/pwm/pwm_byt_platdrv.c
index 899a713..5faab85 100644
--- a/drivers/misc/pwm/pwm_byt_platdrv.c
+++ b/drivers/misc/pwm/pwm_byt_platdrv.c
@@ -111,7 +111,7 @@ static int __init pwm_byt_init_driver(void)
 {
 	return platform_driver_probe(&pwm_byt_plat_driver, pwm_byt_plat_probe);
 }
-module_init(pwm_byt_init_driver);
+subsys_initcall(pwm_byt_init_driver);
 
 static void __exit pwm_byt_exit_driver(void)
 {
-- 
1.7.9.5

